rdr_service/tools/tool_libs/consents.py

```diff
@@ -3,8 +3,10 @@
 from datetime import datetime, timedelta
 from dateutil.parser import parse
 from itertools import islice
+from typing import Collection
 
 import requests
+from sqlalchemy import Boolean, Integer
```

Comment:
Instead of importing these, and leveraging them from sqlalchemy, would it make more sense to use the more pythonic way of testing for this?
`isinstance(value, bool)`
`isinstance(value, int)`

Reply:
SqlAlchemy's `Boolean` and `Integer` are classes of their own rather than labels for the primitive data types. So later in the code we'd need to check if the fields are instances of these rather than the primitive types.

`isinstance(ConsentFile.is_signature_valid.expression.type, bool)` returns false, but `isinstance(ConsentFile.is_signature_valid.expression.type, Boolean)` returns true.